packages/graphql/tests/integration/interfaces/relationships/declare-relationship/interface-simple.int.test.ts

```diff
@@ -1333,7 +1333,7 @@ describe("interface with declared relationships", () => {
             ]
         );
     });
-
+    // FLAKY_TEST
```

Comment:
Is this noted with the intention to work on it in this PR? Otherwise we should probably make a note to work on this separately.

Reply:
I just noticed that this test is flaky and I thought that by adding this comment could help us on keeping track of them!